### ISSUE TITLE
[X] Accessibility: Missing alt text on detail window icons

### ISSUE THREAD

**Issue by mtedeschi** [InteractiveMechanics/qa-training-repo#1]
**Description**
We're missing alt text on icons at the top in the map page detail windows. Reported by WAVE.

**Screenshot**
<img width="350" alt="screen shot 2017-06-15 at 4 05 52 pm" src="https://user-images.githubusercontent.com/842387/27199590-90a9fe70-51e4-11e7-8b6b-89b73c197530.png">
